backend/src/routes/loginRoutes.ts

```diff
@@ -0,0 +1,113 @@
+import cors from "cors";
+import express, { Request, Response, Router } from "express";
+
+const router: Router = express.Router();
+const corsOptions = {
+  origin: ["http://localhost:5173", "https://ccidc.org/"],
```

Comment:
These two links should be in the .env file. 

The .env file does not get pushed to protect the information. So once all the information is put in .env file you then share the contents of the file with use  say through slack